iplass-core/src/main/java/org/iplass/mtp/impl/counter/CachableRdbTableCounterService.java

```diff
@@ -369,8 +369,8 @@ public Object toCacheKey(String cacheKeyString) {
 
 			int index = cacheKeyString.indexOf(':');
 			int tenantId = Integer.parseInt(cacheKeyString.substring(0, index));
-			int index2 = cacheKeyString.indexOf(index + 1, ':');
-			String type = cacheKeyString.substring(index, index2);
+			int index2 = cacheKeyString.indexOf(':', index + 1);

```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer

If the indexOf() search on line 372 does not find the substring and returns -1, the call to substring() on line 373 will throw IndexOutOfBoundsException.